hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -1,3 +1,7 @@
+---
+title: "Analog Sensor Top Technical Specification"

```

Comment:
@zi-v 
this is a minor change that works better with opentitan's document generation.
Let me know if this is okay with you.

Also I guess we should do an doc update for the recently added ports?

Reply:
OK